client/src/components/Modal/Modal.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
+
+.Modal {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 582px;
+    height: 309.39px;
```

Comment:
Rather than set an explicit height and width, can you use percentages and set a the padding?